packages/presentation/lib/src/pages/movie_details_page/widgets/cast_and_crew.dart

```diff
@@ -0,0 +1,178 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_svg/svg.dart';
+import 'package:presentation/generated/l10n.dart';
+import 'package:presentation/src/common_widgets/image_error_widget.dart';
+import 'package:presentation/src/pages/movie_details_page/model/cast_and_crew.dart';
+import 'package:presentation/src/pages/movie_details_page/widgets/movie_details_shadow.dart';
+import 'package:presentation/style/text_styles/styles.dart';
+import 'package:presentation/utils/images_container.dart';
+import 'package:presentation/style/dimens.dart';
+
+const ellipsesInImageButtonCount = 3;
+const shadowListLength = 4;
+
+class CastAndCrewList extends StatelessWidget {
+  const CastAndCrewList({
+    Key? key,
+    required this.castList,
+  }) : super(key: key);
+  final List<CrewAndCastUi> castList;
+
+  @override
+  Widget build(BuildContext context) {
+    return Padding(
+      padding: const EdgeInsets.all(Dimens.padding_18),
+      child: Container(
+        width: double.infinity,
+        child: Column(
+          mainAxisAlignment: MainAxisAlignment.start,
+          crossAxisAlignment: CrossAxisAlignment.start,
+          children: [
+            Row(
+              children: [
+                Text(
+                  S.of(context).castAndCrew,
+                  style: sfProSemiMedium18px,
+                ),
+                Spacer(),
+                TextButton(
+                  onPressed: () {},
```

Comment:
why you use textButton widget if you don't need onPressed
change to text  or  add comment if it needs refactor